start_bs.py

```diff
@@ -105,9 +105,9 @@ def filter_camera_data(camera):
     camera.stats5.read_attrs = ['total', 'centroid']
 
 class SampleXYZ(Device):
-    x = Cpt(EpicsMotor, ':GX}Mtr')
-    y = Cpt(EpicsMotor, ':PY}Mtr')
-    z = Cpt(EpicsMotor, ':PZ}Mtr')
+    x = Cpt(EpicsMotor, ':X}Mtr')
```

Comment:
this seems like a good place to extract a configuration variable - it cannot remain here

Reply:
This code will be different for the MD2. I originally tried to make this part configurable, but did not have the time.